web/templates/right_sidebar.hbs

```diff
@@ -37,11 +37,13 @@
                     {{tooltip_hotkey_hints "?"}}
                 </template>
             </a>
+            {{#if realm_rendered_description}}
             <div class="only-visible-for-spectators">
                 <div class="realm-description">
                     <div class="rendered_markdown">{{rendered_markdown realm_rendered_description }}</div>
                 </div>
             </div>
+            {{/if}}
```

Comment:
Hmm, I'm not sure this is the right fix; the code seems to think it should always be present.
```
function initialize_right_sidebar() {                                                                   
    const rendered_sidebar = render_right_sidebar({                                                     
        can_invite_others_to_realm: settings_data.user_can_invite_others_to_realm(),                    
        realm_rendered_description: page_params.realm_rendered_description,                             
    });                                                                                                 
```

Did this fail for spectators, or for normal users?

Reply:
From `home.py`: 

```
                                                                                                        
    if user_profile is None:                                                                            
        # Get rendered version of realm description which is displayed in right                         
        # sidebar for spectator.                                                                        
        page_params["realm_rendered_description"] = get_realm_rendered_description(realm)               
        page_params["language_cookie_name"] = settings.LANGUAGE_COOKIE_NAME                             
                                                                                                        
```

So I guess for non-spectators? I just want to make sure that this isn't failing to appear for everyone.

Reply:
Right, `realm_rendered_description` is `undefined` for non-spectators. This had no visible effect because it’s inside `<div class="only-visible-for-spectators">` and `clean_user_content_links` happened to ignore it.